src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/price/Price.kt

```diff
@@ -72,6 +83,23 @@ data class Price(
     effectiveYear = effectiveYear,
     addedAt = addedAt
   )
+
+  /**
+   * Duplicate months will be ignored. You must remove first.
```

Comment:
Ignoring duplicates for now.  Not sure but I may change this later e.g. simply replace what is there. Fundamentally though this gives us something to work with for now.